pkg/util/version/const.go

```diff
@@ -12,8 +12,15 @@ import (
 const InstallArchitectureVersion = api.ArchitectureVersionV2
 
 const (
-	DevClusterGenevaLoggingEnvironment   = "Test"
-	DevClusterGenevaLoggingConfigVersion = "2.3"
+	DevClusterGenevaLoggingAccount       = "AROClusterLogs"
+	DevClusterGenevaLoggingConfigVersion = "2.4"
+	DevClusterGenevaLoggingNamespace     = "AROClusterLogs"
+	DevClusterGenevaMetricsAccount       = "AzureRedHatOpenShiftCluster"
+	DevGenevaLoggingEnvironment          = "Test"
```

Comment:
Should `DevGenevaLoggingEnvironment` be consistent with the naming from comments on the `pkg/env/env.go` prod struct? `DevClusterGenevaLoggingEnvironment`?